Simulation/test_file_making.m

```diff
@@ -0,0 +1,6 @@
+%test file making
+filename = "my_test_file2.txt";
+fid = fopen( filename, "w" );
+A = [ 1 2 3 4 5 6 7 8 4 ];
+save (filename,"A");
+fclose ( fid );
```

Comment:
We can remove this file from the pull request since it is a test file and doesn't serve a purpose beyond testing variable saving.